modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -69,9 +69,9 @@
     }
   },
   "paths": {
-    "/power-of-attorney-requests/{id}/decision": {
```

Comment:
why the decision to remove the id from here? just curious.

Reply:
the `{id}` in this case is the `procId` or "process id." imo, it seemed unidiomatic to include it in the URL, which implies the id belongs to a "power of attorney request"-type object. so now we explicitly send the `procId` in the body 🤷